### PR TITLE
Added Footer

### PR DESCRIPTION
**Pull request by Sumandeep-Kaur**
Fixes: #66

### CODE REVIEW

**Review by nandini7637 — APPROVED** [review]

**Comment by nandini7637** [comment]
@Sumandeep-Kaur make a folder of Footer_01 and inside that add code and assets

**Comment by Sumandeep-Kaur** [comment]
> @Sumandeep-Kaur make a folder of Footer_01 and inside that add code and assets

Made the changes, Please have a look!

File: Footers/code/style.css [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+	transition: 0.3s;
+}
+.col ul li {
+	margin: 10px 0;
+	height: 25px;
+}
+.col li i {
+	margin-right: 20px;
+}
+*/
```
Comment (thepranaygupta) [inline Footers/code/style.css]: please remove this commented code.

Comment (thepranaygupta) [inline Footers/code/style.css]: please remove this commented code

**Review by thepranaygupta — CHANGES_REQUESTED** [review]

**Comment by Sumandeep-Kaur** [comment]
Removed! @thepranaygupta

**Pull request merged**